api/Models/ServiceManagement/Entities/ServiceAvailability.cs

```diff
@@ -0,0 +1,13 @@
+using PointOfSale.Models.Shared.Entities;
+
+namespace PointOfSale.Models.ServiceManagement.Entities;
+
+public class ServiceAvailability : EntityBase<int>
+{
+    public required int ServiceId { get; set; }
+    public required int ServiceResourceId { get; set; }
+    public required int Priority { get; set; }
```

Comment:
kas yra service priority?

Reply:
Service priority parodo pagal ka bus priskirtas tam tikras resursas. (Pvz.: jei rezervuojasi staliuka ir yra laisvi dvivieciai ir keturvieciai stalai tai service kai norima rezervuoti vakariene dviems pirmu prioritetu rezervuoja dvvieti, jei ju nera - keturvieti)

Reply:
seems over complicated for mvp but ok